src/main/java/com/microsoft/azure/search/samples/client/SearchIndexClient.java

```diff
@@ -0,0 +1,279 @@
+package com.microsoft.azure.search.samples.client;
```

Comment:
@ruiqiding It looks like you moved files and changed their content at the same time, which makes review very difficult. Please do these in separate commits so it's easier to review.